app/src/main/java/live/hms/app2/ui/meeting/pinnedvideo/VideoListItemDiffUtil.kt

```diff
@@ -17,7 +17,7 @@ class VideoListItemDiffUtil(
   override fun getNewListSize() = newList.size
 
   override fun areItemsTheSame(oldItemPosition: Int, newItemPosition: Int): Boolean {
-    return oldList[oldItemPosition].id == newList[newItemPosition].id
+    return oldList[oldItemPosition].track.peer.peerID == newList[newItemPosition].track.peer.peerID
```

Comment:
This would mean the regular video and screenshare track of a peer will be considered same. Is this what we want?